src/sentry/replays/endpoints/organization_replay_selector_index.py

```diff
@@ -336,9 +338,9 @@ def make_selector_name(row) -> str:
                 "tag": row["click_tag"],
                 "testid": row["click_testid"],
                 "title": row["click_title"],
+                "component_name": row["click_component_name"],
             },
             "project_id": row["project_id"],
-            "component_name": row["click_component_name"],
```

Comment:
Do we have frontend components that rely on this old response structure? If so we may want make sure the frontend is back/forwards compatible

Reply:
no we don't! this was added in recently by me for including react component names in selectors